packages/flutter/lib/src/rendering/custom_layout.dart

```diff
@@ -91,6 +91,9 @@ abstract class MultiChildLayoutDelegate {
     }
   }
 
+  /// Override this method to return true when the children need to be laid out.
+  bool shouldRelayout() => false;
```

Comment:
This function should be passed the oldDelegate so that it can make a meaningful decision.